Core/OxidEE_Events.php

```diff
@@ -20,7 +21,6 @@
  */
 class OxidEE_Events
```

Comment:
One thing we do in the class is using `oxDb` - that's an outdated way of accessing the database. can you also refactor it with using `\OxidEsales\Eshop\Core\DatabaseProvider::getDb()`!?

https://docs.oxid-esales.com/developer/en/6.0/update/eshop_from_53_to_6/modules.html